prime-router/settings/prod/0018-nj-receivers.yml

```
@@ -0,0 +1,54 @@
+---
+- name: nj-doh
+  description: New Jersey Department of Health
+  jurisdiction: STATE
+  stateCode: NJ
+  receivers:
+    - name: elr
+      organizationName: nj-doh
+      topic: covid-19
+      jurisdictionalFilter:
+        - orEquals(ordering_facility_state, NJ, patient_state, NJ)
+        - matches(processing_mode_code, P)
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        receivingApplicationName: ELRS
+        receivingApplicationOID: 2.16.840.1.113883.3.1299.5.1.6.1
+        receivingFacilityName: NJDOH
+        receivingFacilityOID: 2.16.840.1.113883.3.1299
+        truncateHDNamespaceIds: true
+      transport:
+        type: SFTP
+        host: njgov.moveitcloud.com
+        port: 22
+        filePath: ./PROD
+      timing:
+        operation: MERGE
+        numberPerDay: 12
+        initialTime: 01:15
```

```suggestion
        initialTime: 01:30
```
Let's start shifting these times slightly +/-, so not all 60 states are waking up at the same time.